IDEAS/Examples/Tutorial/DetailedHouse/DetailedHouse5.mo

```diff
@@ -78,14 +78,51 @@ First implementation for the IDEAS crash course.
 </html>", info="<html>
 <p>
 This example demonstrates the use of the RectangularZoneTemplate.
-The one-zone implementation with one window of example 1 is repeated here as 
+The one-zone implementation with one window of <a href=\"modelica://IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse1\">
+IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse1</a> is repeated here as 
 a two-zone implementation with two windows that are north and south oriented.
 Note the different temperature responses of the zones. 
 </p>
-</html>"), experiment(
+<h4>Required models</h4>
+<ul>
+<li>
+<a href=\"modelica://IDEAS.BoundaryConditions.SimInfoManager\">
+IDEAS.BoundaryConditions.SimInfoManager</a>
+</li>
+<li>
+<a href=\"modelica://IDEAS.Buildings.Components.RectangularZoneTemplate\">
+IDEAS.Buildings.Components.RectangularZoneTemplate</a>
+</li>
+<li>
+Construction and glazing records from <a href=\"modelica://IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse1\">
+IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse1</a>
+</li>
+</ul>
+<h4>Connection instructions</h4>
+<p>
+The model consists of two <code>RectangularZoneTemplates</code> and a <code>SimInfoManager</code>. The required parameters are set in 
+the templates, with careful attention to all tabs. The internal wall is defined in only one of the two 
+templates, while an <i>external connection</i> is used for the other template. The <code>InternalWall</code> and 
+<code>External</code> options cause a yellow bus connector to appear on each template, which must then be connected to each other.
+</p>
+<h4>Reference result</h4>
+<p>
+The figure bellow shows the zone temperatures of both zones. Note the large influence that the
+window placement has on the zone dynamics!
+</p>
+<p align=\"center\">
+<img alt=\"Zone temperature for the zone with the north oriented window (blue) and the zone with the south
+oriented window (red).\"
```

Comment:
This information is not visible in the documentation view. Consider copying/moving this (and all uses of `img alt` in previous and future models) to the Reference results section.